pkg/scripts/csv-to-couchdb.ts

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env -S deno run --allow-read --allow-net
+
+import Denomander from "https://deno.land/x/denomander@0.9.1/mod.ts";
+import { readCSV } from "https://deno.land/x/csv/mod.ts";
+import ProgressBar from "https://deno.land/x/progress@v1.2.4/mod.ts";
+import { BufReader } from "https://deno.land/std/io/buffer.ts";
+import { bulkDocs, createDatabase } from "../couchdb/couchdb.ts"
+
+/**
+  * Counts lines in a csv file for progress bar calculations
+  * @param {string} filePath - csv file path 
+  * @return {Number} - Number of lines in file
+  */
+
+const howManyLines = async function (filePath: string) {
+    let lineCount: number = 0;
+    var file;
+    console.log({ filePath })
+    try {
+        file = await Deno.open(filePath);
+        const bufReader = new BufReader(file);
+        while (await bufReader.readLine()) {
+            lineCount++
+        };
+    } finally {
+        if (file) file.close();
+    }
+    return lineCount;
+}
+
+//#region progress bar
+/**
+  * Renders progress bar
+  * @param {ProgressBar} progress - progress bar instance 
+  * @param {Number} progress - chunk size for incrementation
+  */
+let completed = 0;
+function downloading(progress: ProgressBar, chunkSize: number) {
```

Comment:
This function prints a progress bar. It does this while the CSV file is being processed. There is no download going on here. A better name could be `printProgress`. It could take `completed` as a parameter, instead of `chunkSize`.